src/mqtt.rs

```diff
@@ -1,9 +1,12 @@
 use std::{thread, time::Duration};
```

Comment:
in light of the _output channel_ feature (see below) it probably makes sense to rename this file to `home_assistent.rs` since it will implement the output channel to a HA instance.